### PR TITLE
chore(TVOS): rebuild TVOSExample app with RN 0.74.1

### PR DESCRIPTION
**Pull request by alduzy**
## Description

PR rebuilding TVOSExample from scratch with RN 0.74.1.

## Changes

- rebuild TVOSExample app with latest RN version

## Checklist

- [ ] Included code example that can be used to test this change
- [ ] Updated TS types
- [ ] Updated documentation: 
  - [ ] https://github.com/software-mansion/react-native-screens/blob/main/guides/GUIDE_FOR_LIBRARY_AUTHORS.md
  - [ ] https://github.com/software-mansion/react-native-screens/blob/main/native-stack/README.md
  - [ ] https://github.com/software-mansion/react-native-screens/blob/main/src/types.tsx
  - [ ] https://github.com/software-mansion/react-native-screens/blob/main/src/native-stack/types.tsx
- [ ] Ensured that CI passes

### CODE REVIEW

**Review by kkafar — COMMENTED** [review]
I guess we need to make CI work before we proceed here. It does somehow build on main, wo we should be able to make it work here too (as well as running it locally).

File: TVOSExample/ios/Podfile [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
+#     # An absolute path to your application root.
+#     :app_path => "#{Pod::Config.instance.installation_root}/.."
+#   )
+# 
+#   target 'TVOSExampleTests' do
+#     inherit! :complete
+#     # Pods for testing
+#   end
+# 
+# end
```
Comment (tboba) [inline TVOSExample/ios/Podfile:34]: Should this part be commented out?

File: TVOSExample/src/BottomTabsExample.tsx
Comment (tboba) [inline TVOSExample/src/BottomTabsExample.tsx:1]: Can we grab the examples from `apps` directory?

**Review by tboba — CHANGES_REQUESTED** [review]
Great job! 🥳

File: TVOSExample/src/BottomTabsExample.tsx
Comment (alduzy) [reply TVOSExample/src/BottomTabsExample.tsx:1]: I created a separate PR for that: https://github.com/software-mansion/react-native-screens/pull/2173

File: TVOSExample/android/app/src/main/res/drawable/tv_banner.png
[the remark this answers is not in the document]
Comment (alduzy) [reply TVOSExample/android/app/src/main/res/drawable/tv_banner.png:1]: Yes 😄 https://github.com/react-native-tvos/react-native-tvos/blob/tvos-v0.74.0/packages/react-native/template/android/app/src/main/res/drawable/tv_banner.png

File: TVOSExample/android/app/build.gradle [hunk 2]
```diff
@@ -1,150 +1,87 @@
...
+    //   A list of extra flags to pass to the 'bundle' commands.
+    //   See https://github.com/react-native-community/cli/blob/main/docs/commands.md#bundle
+    // extraPackagerArgs = []
+
+    /* Hermes Commands */
+    //   The hermes compiler command to run. By default it is 'hermesc'
+    // hermesCommand = "$rootDir/my-custom-hermesc/bin/hermesc"
+    //
+    //   The list of flags to pass to the Hermes compiler. By default is "-O", "-output-source-map"
+    // hermesFlags = ["-O", "-output-source-map"]
```
[the remark this answers is not in the document]
Comment (alduzy) [reply TVOSExample/android/app/build.gradle:51]: Yes, by default at least: https://github.com/react-native-tvos/react-native-tvos/blob/tvos-v0.74.0/packages/react-native/template/android/app/build.gradle

File: .github/workflows/tv-os-build-test.yml [hunk 3]
```diff
@@ -32,6 +32,10 @@ jobs:
           node-version: 18
           cache: 'yarn'
           cache-dependency-path: 'TVOSExample/yarn.lock'
+      - name: Use latest stable Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
```
[the remark this answers is not in the document]
Comment (alduzy) [reply .github/workflows/tv-os-build-test.yml]: https://github.com/software-mansion/react-native-screens/pull/2155/commits/b56ee31761ef41f0f136e59802404bccfe4f800f

File: .github/workflows/tv-os-build-test.yml [hunk 4]
```diff
@@ -40,4 +44,4 @@ jobs:
         run: pod install
       - name: Build app
         working-directory: ${{ env.WORKING_DIRECTORY }}
-        run: yarn tv-os
+        run: npx react-native run-ios --scheme="TVOSExample-tvOS" --simulator="Apple TV"
```
[the remark this answers is not in the document]
Comment (alduzy) [reply .github/workflows/tv-os-build-test.yml:43]: Unfortunately I could not make the expo script to work properly in CI

**Review by tboba — APPROVED** [review]
Okay then 👍

**Pull request merged**